include/common_structs.h

```diff
@@ -391,4 +391,8 @@ typedef struct {
     f32* unk_00[5];
 } struct_D_800E261C;
 
+typedef struct {
+    s32 playerRelatedFlags[8]; // Set to 1 when acceleration is pressed for each player
```

Comment:
I'm not really a fan of this name and then having the comment being more specific. I'll take a closer look when I have some more time.

Reply:
I've done some more testing and I don't think those other bytes get ever written to, and they always stay 0. I think the game literally uses the whole 4 bytes for this one flag, which is weird, but there are weirder things in the code. I changed it from a struct to an s32.